performance-tests/docker-compose.yaml

```diff
@@ -68,9 +67,23 @@ services:
       timeout: 10s
     ports:
       - "9092"
+    environment:
+      TEST_NAME: "${TEST_NAME:-manual}"
+      "ASYNC_PROFILER_FLAGS": "start,cpu,flamegraph,file=/tmp/profiler-results/%{TEST_NAME}/flamegraph-%t.html"
+    configs:
+      - source: proxy-config
+        target: /opt/kroxylicious/config/config.yaml
     volumes:
-      - ./${KROXYLICIOUS_CONFIG:-unused.yaml}:/opt/kroxylicious/config/config.yaml
+      - type: "bind"
+        source: "${PROFILING_OUTPUT_DIRECTORY:-/tmp/profiler-results/}"
+        target: "/tmp/profiler-results/"
+        read_only: false
+
     networks:
       - perf_network
     depends_on:
       - kafka
+
+configs:
+  proxy-config:
+    file: "proxy-config.yaml"
```

Comment:
Moving to a `configs` node gives us more flexibility going forward if/when we get to dynamic reloading the this will just work.